src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,118 @@
+.page__cards {
+  margin-top: 50px;
+}
+
+.cards {
+  @include flex();
+  flex-wrap: wrap;
+
+  &__container {
+    width: cardWidth($cardsWidth, $cardsCardMargin);
+    @include flex();
+    flex-wrap: wrap;
+    
+  }
+
+  &__card {
+    margin-left: $cardsCardMargin;
+    margin-right: $cardsCardMargin;
+    margin-bottom: 50px;
+  }
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
```

Comment:
add align-items: center to center all elements horizontally
in such case you no longer need horizontal margins for image, for example